provisioner/clusterpy.go

```diff
@@ -35,30 +35,31 @@ import (
 )
 
 const (
-	providerID                     = "zalando-aws"
-	etcdStackFileName              = "etcd-cluster.yaml"
-	clusterStackFileName           = "cluster.yaml"
-	defaultNamespace               = "default"
-	kubectlNotFound                = "(NotFound)"
-	tagNameKubernetesClusterPrefix = "kubernetes.io/cluster/"
-	subnetELBRoleTagName           = "kubernetes.io/role/elb"
-	resourceLifecycleShared        = "shared"
-	resourceLifecycleOwned         = "owned"
-	mainStackTagKey                = "cluster-lifecycle-controller.zalando.org/main-stack"
-	stackTagValueTrue              = "true"
-	subnetsConfigItemKey           = "subnets"
-	subnetsValueKey                = "subnets"
-	availabilityZonesConfigItemKey = "availability_zones"
-	availabilityZonesValueKey      = "availability_zones"
-	vpcIDConfigItemKey             = "vpc_id"
-	subnetAllAZName                = "*"
-	maxApplyRetries                = 10
-	configKeyUpdateStrategy        = "update_strategy"
-	updateStrategyRolling          = "rolling"
-	updateStrategyCLC              = "clc"
-	defaultMaxRetryTime            = 5 * time.Minute
-	clcPollingInterval             = 10 * time.Second
-	clusterStackOutputKey          = "ClusterStackOutputs"
+	providerID                         = "zalando-aws"
+	etcdStackFileName                  = "etcd-cluster.yaml"
+	clusterStackFileName               = "cluster.yaml"
+	defaultNamespace                   = "default"
+	kubectlNotFound                    = "(NotFound)"
+	tagNameKubernetesClusterPrefix     = "kubernetes.io/cluster/"
+	subnetELBRoleTagName               = "kubernetes.io/role/elb"
+	resourceLifecycleShared            = "shared"
+	resourceLifecycleOwned             = "owned"
+	mainStackTagKey                    = "cluster-lifecycle-controller.zalando.org/main-stack"
+	stackTagValueTrue                  = "true"
+	subnetsConfigItemKey               = "subnets"
+	subnetsValueKey                    = "subnets"
+	availabilityZonesConfigItemKey     = "availability_zones"
+	availabilityZonesValueKey          = "availability_zones"
+	vpcIDConfigItemKey                 = "vpc_id"
+	subnetAllAZName                    = "*"
+	maxApplyRetries                    = 10
+	configKeyUpdateStrategy            = "update_strategy"
+	updateStrategyRolling              = "rolling"
+	updateStrategyCLC                  = "clc"
+	defaultMaxRetryTime                = 5 * time.Minute
+	clcPollingInterval                 = 10 * time.Second
+	clusterStackOutputKey              = "ClusterStackOutputs"
+	decommissionNodeNoScheduleTaintKey = "clm_decommission_node_no_schedule_taint"
```

Comment:
Let's drop the `clm_` part? It shouldn't matter which application does that, and if we ever move it to CLC it'll just be more confusing.

Reply:
done